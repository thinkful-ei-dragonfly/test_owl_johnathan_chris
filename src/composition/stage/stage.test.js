import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Stage from './stage';

describe('User component', () => {
    it('renders without crashing', () => {
        const div = document.createElement('div');
        ReactDOM.render(<Stage />, div);
        ReactDOM.unmountComponentAtNode(div);
      });

    it('renders the UI as expected', () => {
    const tree = renderer
      .create(<Stage name="Johnathan" avatar='https://robohash.org/doloredolorescupiditate.jpg?size=200x200&set=set1'/>)
      .toJSON();
    expect(tree).toMatchSnapshot();  
  });
  });