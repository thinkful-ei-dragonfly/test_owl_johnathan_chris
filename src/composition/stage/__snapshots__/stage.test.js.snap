// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`User component renders the UI as expected 1`] = `
<div
  className="stage-container"
>
  <span
    className="user-name"
  >
    Johnathan
  </span>
  <img
    alt="stage-avatar"
    src="https://robohash.org/doloredolorescupiditate.jpg?size=200x200&set=set1"
  />
  <span>
    <button
      className="volume"
    >
      ðŸ”Š
    </button>
  </span>
</div>
`;
